src/test/java/org/monarchinitiative/biodownload/FileDownloaderImplTest.java

```diff
@@ -0,0 +1,29 @@
+package org.monarchinitiative.biodownload;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+
+public class FileDownloaderImplTest {
+
+    private IBioDownloader bioDownloader;
+
+    @BeforeEach
+    void init() {
```

Comment:
This should be `public` too, otherwise we may run into issues with inaccessibility of the method after we switch to JPMS (Jigsaw).

Reply:
resolved in 5c6e06a